index.html

```diff
@@ -0,0 +1,148 @@
+                    <li><a href="/about.html">Schedule</a></li>
+                    <li><a href="/about.html">Join</a></li>
+                    <li><a href="/about.html">Sponsor</a></li>
+                    <li><a href="/about.html">News</a></li>
+                    <li><a href="/about.html">CC Campaign</a></li>
+                </ul>
+            </nav>
+            <a href="#"><img src="images/header-logo.png" alt="Global Summit Logo" class="mobile-header-logo"></a>
+        </div>
+
+        <ul class="navbar-banner">
+            <li><a href="#"><img src="images/facebook-logo.png" class="mini-navbar-img" alt="facebook logo"></a></li>
+            <li><a href="https://twitter.com/AyokunnumiA"><img src="images/twitter-logo.png" class="mini-navbar-img"
+                        alt="twitter logo"></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Logo</a></li>
+            <li><a href="#">Logout</a></li>
+        </ul>
+
+        <div>
+            <nav class="desktop-navbar">
+                <div><img src="images/header-logo.png" alt=""></div>
+                <ul class="desktop-navbar-list">
+                    <li><a href="/about.html" class="desktop-navbar-link">About</a></li>
+                    <li><a href="#" class="desktop-navbar-link">Program</a></li>
+                    <li><a href="#" class="desktop-navbar-link">Join</a></li>
+                    <li><a href="#" class="desktop-navbar-link">Sponsor</a></li>
+                    <li><a href="#" class="desktop-navbar-link">News</a></li>
+                    <li><a href="#" class="desktop-navbar-link CC-Campaign-link">CC Campaign</a></li>
+                </ul>
+            </nav>
+        </div>
+
+        <div class="desktop-header-section">
+            <div class="mobile-header-section">
+                <p class="header-theme">"You Never Fail Until You Stop Trying"</p>
+                <h1 class="h1-desktop">Content Creator <br> Global Summit 2023</h1>
+                <p class="header-hero-desc">A joyful celebration believing in the value of openness and
+                    sharing, creating a positive change with people from over 80
+                    countries is taking place in October, in Korea.
+                </p>
+                <div class="date-time">
+                    <time>2023-10-15(Thurs) ~ 16(Fri)</time>
+                    <p class="venue">&#x40;National Museum of Lagos, Muson Center Lagos and more</p>
+                </div>
+            </div>
+        </div>
+    </header>
+    <main>
+        <section class="main-program-container">
+            <div>
+                <h2>Main Program</h2>
+                <span class="line"></span>
+            </div>
+            <div class="main-program-content">
+                <div class="main-program-short">
+                    <div class="main-program-mini">
+                        <img src="images/lecture-img.png" alt="">
```

Comment:
- [ ] You did a good job working on the previous reviewer's suggestions! 💯 However, there's still some room to improve the quality of your changes:

| Template |
|---|
| ![image](https://github.com/Ayokunnumi1/Conference-Page/assets/77069011/a4b3c44e-43bf-46bf-932b-f132b03becb3) |

| Your design |
|---|
| ![image](https://github.com/Ayokunnumi1/Conference-Page/assets/77069011/df5cba62-a1ee-4b47-8c65-b5e61b8bad89) |

If you see, the spacing is not the same as shown in the template, this visual bug got introduced in the latest changes you made. Please fix it! 🤓 🔧